workflow/tests/base_results/results_bills.csv

```diff
@@ -0,0 +1,330 @@
+HPXML,Electricity: Fixed ($),Electricity: Marginal ($),Electricity: PV Credit ($),Electricity: Total ($),Natural Gas: Fixed ($),Natural Gas: Marginal ($),Natural Gas: Total ($),Fuel Oil: Total ($),Propane: Total ($),Wood Cord: Total ($),Wood Pellets: Total ($),Coal: Total ($),Total ($)
```

Comment:
👍 Great to have this file.

I assume it reflects the order of rows in the individual results_bills.csv files? If so, please put the total at the top of the file, not the bottom, like we do in results_annual.csv.